lib/contentful-util.js

```diff
@@ -2,7 +2,7 @@ const { documentToHtmlString } = require('@contentful/rich-text-html-renderer');
 const pkg = require('../package.json');
 
 function normalizeEntries({ contentTypes, entries, defaultLocaleCode, options }) {
-    return entries.map(entry => normalizeEntry({ contentTypes, entry, defaultLocaleCode, options })).filter(entry => entry);
+    return entries.map(entry => normalizeEntry({ contentTypes, entry, defaultLocaleCode, options })).filter(Boolean).filter(entry => entry);
```

Comment:
maybe better to delete the `.filter(entry => entry)` in favor of `.filter(Boolean)` then?
I think the `.filter(entry => entry)` had the same intention.

Reply:
Ah yes, I was wondering about this myself. I thought `.filter(entry => entry)` would have had the same effect, but it seems to be letting the nulls through. I wanted to do some more testing, but the filter(Boolean) seemed to do the trick for now.